docs/source/installation.rst

```diff
@@ -35,4 +35,61 @@ The latest release is |release|.
 
     pip install git+https://github.com/mesosphere/dcos-e2e.git@master
 
+
+Windows
+~~~~~~~
+
```

Comment:
Please document that this gets you `master` in the VM.

There are trade offs and various options. From what I can see those options are:

* No dcos-e2e installation. You install your preferred version yourself.
* master
* The latest released version

Let's at least document the choice here, and note to ourselves that we might want to change this. I find it a little strange that the Homebrew recipe gets us the latest released version, and this gets us "master".

Reply:
@adamtheturtle Do you manually update that for each release? Are there release instructions to manage this?

Reply:
@jongiddy the release script in the `admin` directory updates the Homebrew recipe.

Reply:
Feel free to ignore this / do it as a follow-up as you see fit.

Reply:
It is done - it will now use the latest release (at the time of downloading the Vagrantfile) and allows overriding using an environment variable.